test/new_relic/gemspec_files_test.rb

```diff
@@ -6,15 +6,13 @@
 
 class GemspecFilesTest < Minitest::Test
   def test_the_test_agent_helper_is_shipped_in_the_gem_files
-    skip if defined?(Rails::VERSION)
-
-    agent_helper_file = 'test/agent_helper.rb'
+    skip if defined?(Rails::VERSION) || File.basename(Dir.pwd).match?('rails')
```

Comment:
@fallwith  - If we skip running this test in the `norails` environment, when does this test run?

Reply:
@kaylareopelle Ugh. Apparently we only run `bundle exec rake test` locally and via the automated CI it's always in `env`. Will address.

Reply:
My understanding is that `norails` is the default environment that gets run when we execute `bundle exec rake test` locally.

Reply:
That might not be the case though, haven't looked in a while.

Reply:
Addressed with [7fcd753](https://github.com/newrelic/newrelic-ruby-agent/pull/2130/commits/7fcd753b89d7f39ca8904a1450c08a3d9a97deeb)

Reply:
I was able to repro locally with `bundle exec rake test:env[norails]` while having it work fine with `bundle exec rake test`.

I believe `bundle exec rake test` will preserve PWD as the agent root, but all of the `env` tests will chdir to `test/environments/<name>`, bundle from there, and then execute from there.

That key difference seemed to be the problem. Now the new test should work in and out of `env`.

We could even allow the test to run on the Rails envs, but there's not much value in having the test run more than once.